cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesBoundary.java

```diff
@@ -98,7 +119,7 @@ public double angleAtBoundary(String node) {
         return nodesVoltage.containsKey(node) ? nodesVoltage.get(node).angle : Double.NaN;
     }
 
-    public List<PropertyBag> equipmentAtNode(String node) {
```

Comment:
This method was public so please deprecate it

Reply:
Done